apps/src/templates/sectionProgress/progressTables/ProgressTableLessonNumber.jsx

```diff
@@ -90,11 +103,16 @@ class ProgressTableLessonNumber extends React.Component {
       >
         {this.renderTooltip()}
         {lockable ? <FontAwesome icon="lock" /> : number}
-        {includeArrow && <span style={styles.line} />}
-        {includeArrow && <span style={styles.arrow} />}
+        {includeArrow && <LessonArrow />}
       </div>
     );
   }
 }
 
+export let ExportedLessonArrow;
+
+if (IN_UNIT_TEST) {
```

Comment:
Is the reason that we need `IN_UNIT_TEST` to prevent this component from being used in other components outside of a unit test?  (I'm asking because this doesn't seem to be widely used and I'm wondering if we want to increase its usage?)

Reply:
No, currently (as far as I know) this arrow is only used in this component, so it doesn't need to be exported except for testing. I would be ok with making it a separate component? Do we have a spot for general shared components? Or maybe I shouldn't write it as a function, and instead identify it for tests with a class name. This was just a pattern that Charlie and I were trying out.

Reply:
`IN_UNIT_TEST` came out of our discussion here https://github.com/code-dot-org/code-dot-org/pull/38565#pullrequestreview-567786931

Reply:
Sorry, my first comment was confusing.  I meant that `IN_UNIT_TEST` doesn't appear to be widely used and I'm wondering if we should increase its usage.

Here's a SO thread I found on this topic: https://stackoverflow.com/questions/54116070/how-can-i-unit-test-non-exported-functions

I kind of like the approach of exporting a single object that contains all the things that need to be exported just for unit testing.  (The SO answer calls it `testables` but I might name it `unitTestExports`.)

I'm ok with either approach.  Maybe also drop a message in #test-tactics with what you choose and see if people have feedback.

Reply:
@jamescodeorg i used that object approach in my PR and called it `subComps` -- `unitTestExports` sounds like a better convention because 1) it's clearer and 2) it can be used more generally for things that aren't subcomponents.

Reply:
@cforkish @jamescodeorg I've updated the export to match `testable` pattern described by the stackoverflow, let me know what you think.

Reply:
I think Charlie and I both preferred the name `unitTestExports` instead of `testables`.  Would you be ok with that name?

Reply:
That sounds good!